src/platforms/esp32/main/main.c

```diff
@@ -136,3 +149,33 @@ void app_main()
         }
     }
 }
+
+#ifdef USE_USB_SERIAL
+void init_usb_serial()
+{
+    /* Setting TinyUSB up */
+    ESP_LOGI(TAG, "USB initialization");
```

Comment:
This line will never be printed (since the console is not yet available) but there is probably no harm in leaving it in.